alfalfa_worker/__main__.py

```diff
@@ -23,34 +23,29 @@
 #  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ########################################################################################################################
 
-print("Starting Alfalfa Worker")
+print("Starting Alfalfa Dispatcher")
 
-import os
 import sys
 import traceback
 
 # Determine which worker to load based on the QUEUE.
 # This may be temporary for now, not sure on how else
 # to determine which worker gets launched
-if 'local-queue1' in os.environ.get('JOB_QUEUE_URL', None):
-    from alfalfa_worker.worker_openstudio.worker import WorkerOpenStudio as Worker
-elif 'local-queue2' in os.environ.get('JOB_QUEUE_URL', None):
-    from alfalfa_worker.worker_fmu.worker import WorkerFmu as Worker
-else:
-    print(f"Unknown queue in env var of JOB_QUEUE_URL with {os.environ.get('JOB_QUEUE_URL', None)}")
+from alfalfa_worker.dispatcher import Dispatcher
+
 
 if __name__ == '__main__':
     try:
-        worker = Worker()
-        worker.worker_logger.logger.info("Worker initialized")
+        dispatcher = Dispatcher()
```

Comment:
The dispatcher is now in charge of handing the simulation to the appropriate worker runner code. Worker is a misnomer a bit because it isn't a separate node/pod.